internal/build/helper.go

```diff
@@ -0,0 +1,158 @@
+package build
```

Comment:
Can we have some tests for this file and these helper functions?

Reply:
yess sure:)

Reply:
hey @dlion added the tests, sorry for getting late!